installer/x86_64/install.sh

```diff
@@ -20,6 +20,25 @@ _trap_push() {
 }
 _trap_push true
 
+read_conf_file() {
```

Comment:
>read_conf_file [](start = 0, length = 14)

Could you write a unit test to cover old cases and your new feature?

Reply:
Sure. But what kind of the test will be better and suitable?
Add a test inside sonic-mgmt and be independent from building. Or, add additional process to test this during building.
I haven't found similar case of this type of unit test.

Reply:
`read_conf_file()` is complex. I expect you have tested it locally. Could you just automate the test in another shell script or a python test runner.
In future we may either improve the code or the testcases.

---
In reply to: [568342160](https://github.com/Azure/sonic-buildimage/pull/6600#discussion_r568342160) [](ancestors = 568342160)

Reply:
I just copied the function to another script and test it with a sample `machine.conf`.
The test can only be triggered manually for now.